Client/ClientGame.js

```diff
@@ -337,13 +339,13 @@ InputListener.prototype.update = function (delta) {
 	                yFac : y_factor,
 	                attack: isAttacking,
 	                inputSeq: this.inputSeq,
-			        direction: direction,
+			        x_direction: direction,
                     itemId: itemId,
                     size : this.player.getSize()
 			     };
 			   if(isAttacking && (numShots>0)){
-			    var tempitem = new Item(this.player.physicsComponent.position.x+60*direction, this.player.physicsComponent.position.y,10,10,itemId, true);
-                Matter.Body.setVelocity( tempitem.physicsComponent,Matter.Vector.create(direction*15,0));
+			    var tempitem = new Item(this.player.physicsComponent.position.x+60*x_direction, this.player.physicsComponent.position.y*(y_direction)sudo,10,10,itemId, true);
```

Comment:
sudo is a typo. I thought I took that out, I mean I ran and tested before the pull request. y_direction is for the up/down shooting